libc/src/unistd/linux/CMakeLists.txt

```diff
@@ -273,6 +287,33 @@ add_entrypoint_object(
     libc.src.errno.errno
 )
 
+add_entrypoint_object(
+  pathconf
+  SRCS
+  pathconf.cpp
+  HDRS
+    ../pathconf.h
+  DEPENDS
+    libc.include.unistd
+    libc.include.sys_syscall
+    libc.src.__support.OSUtil.osutil
+    libc.src.errno.errno
+    libc.src.unistd.pathconf_utils
+)
+
+add_entrypoint_object(
```

Comment:
use `add_object_library/add_header_library` instead since `pathconf_utils` is not an entrypoint.